javascript/HtmlEditorField.js

```diff
@@ -1158,8 +1158,17 @@ ss.editorWrappers['default'] = ss.editorWrappers.tinyMCE;
 				var urlField = this.getURLField(), container = this.closest('.CompositeField'), form = this.closest('form');
 
 				if (urlField.validate()) {
+
+					var val = urlField.val(), orig = val;
+
 					container.addClass('loading');
-					form.showFileView('http://' + urlField.val()).done(function() {
+
+					// add "http://" if a protocol is missing from the url
+					if (val.match(/^https?:\/\//i) == void 0) {
```

Comment:
Well the thinking is now that if you can link to a https resource you should - so if you're going to use `//` you would be better off enforcing `https://`.

I'm guessing in the current state of this PR scheme-less URLs would have `http://` added to give you `http:////`? So maybe what we should do instead is at least detect scheme-less URL input here and upgrade it to the `https://`  if found?